README.md

```diff
@@ -1 +1,24 @@
-# hw01
\ No newline at end of file
+# hw01
+---
+##  Wendy's homework 01 
+---
+  Hi guys, I'm Wendy Chen, a first year PhD student from PME. I'm looking forward to utilizing R to statically analyze and summarize data obtained from my lab. 
+  
+  I don't install my Git at the default location at first so I fail to connect a local RStudio project to a GitHub repo. I uninstall the Git after that, and install it correctly. But it shows *fatal: remote origin already exists*, when I mistype the url command at the beginning.
+  
+  What I find is that "origin" is the default name Git gives to the server I'm using. Because I'm cloning my repository, I need to link *origin* to my url correctly.
+  
+  [The website to know git remote](https://git-scm.com/book/en/v2/Git-Basics-Working-with-Remotes)
+  
+  
+   I open a new project, submit the right url and get the cloning done successfully.
+  ![image of success](https://fiverr-res.cloudinary.com/images/q_auto,f_auto/gigs/128537975/original/5958e62b86a1661db95e8769b149b71161e4a7b2/send-you-encouraging-message-everyday.jpg)
```

Comment:
Awwww ❤️ 💟  I understand your feeling. The first time I learn about git/github, I was very confused, and working with them was very challenging at first 😭